tm2/pkg/bft/abci/types/types.go

```diff
@@ -116,6 +117,25 @@ func (r ResponseBase) IsErr() bool {
 	return r.Error != nil
 }
 
+func (r ResponseBase) EventString() string {
+	var builder strings.Builder
+
+	builder.WriteString("[")
```

Comment:
```suggestion
	builder.WriteString("{")
```

what about generating valid JSON, here?

Reply:
To contain multiple events, I made it output JSON as an array type, but I will modify it.

Reply:
I modified that if there is only one `Event`, it uses curly braces. Also, if there multiple Events are exists, then the outmost part is wrapped with `[`. 

Additionally, to create a valid JSON string, I wrappep string-type data in double quotes using `strconv.Quote`.